src/libasr/codegen/asr_to_c.cpp

```diff
@@ -585,6 +585,7 @@ R"(
 #include <stdio.h>
 #include <string.h>
 #include <lfortran_intrinsics.h>
+#include <math.h>
```

Comment:
I think this should be only included when needed, I think there is a mechanism for that, based on the intrinsic function.